plugins/optimization-detective/detection.php

```diff
@@ -70,8 +70,22 @@ function od_get_cache_purge_post_id(): ?int {
  * @param OD_URL_Metric_Group_Collection $group_collection URL Metric group collection.
  */
 function od_get_detection_script( string $slug, OD_URL_Metric_Group_Collection $group_collection ): string {
+	$use_attribution_build = WP_DEBUG || wp_is_development_mode( 'plugin' );
+
+	/**
+	 * Filters whether to use the web-vitals.js build with attribution.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param bool $use_attribution_build Whether to use the attribution build.
+	 */
+	$use_attribution_build = (bool) apply_filters( 'od_use_web_vitals_attribution_build', $use_attribution_build );
```

Comment:
I don't think the attribution build should be conditionally enabled by default. The reason is that if someone is developing an extension which depends on the attribution build, then they would easily forget to add the filter to enable it.

```suggestion

	/**
	 * Filters whether to use the web-vitals.js build with attribution.
	 *
	 * @since n.e.x.t
	 *
	 * @param bool $use_attribution_build Whether to use the attribution build.
	 */
	$use_attribution_build = (bool) apply_filters( 'od_use_web_vitals_attribution_build', false );
```